orc8r/tools/packer/vagrant-box-upload.sh

```diff
@@ -20,8 +20,9 @@ fi
 
 vagrant cloud auth login --token "$VAGRANT_CLOUD_TOKEN"
 vagrant cloud publish \
-    "${USER}/${BOX}"
+    --release \
+    --force \
```

Comment:
is that necessary? without force one probably has to press `y` or so to continue? feels like a good behaviour